guides/doc-Upgrading_Project/master.adoc

```diff
@@ -36,10 +36,8 @@ endif::[]
 // Performing Post-Upgrade Tasks
 include::common/modules/proc_performing-post-upgrade-tasks.adoc[leveloffset=+2]
 
-ifdef::foreman-el,katello,satellite,orcharhino[]
 // Upgrading Smart Proxy Server
 include::common/modules/proc_upgrading-smartproxy-server.adoc[leveloffset=+2]
```

Comment:
The "Installing_Proxy" guide is marked as "not ready" for Debian: https://docs.theforeman.org/nightly/Installing_Proxy/index-foreman-deb.html
So upgrading w/o installing seems complicated?
That said, the steps look correct if people used the installer for the installation.